kod/object/item/passitem/weapon.kod

```diff
@@ -54,6 +54,7 @@ constants:
    WEAPON_RANGE_HIGH = 3
 
    % Quality mods to number of hits
+   % Deprecated
```

Comment:
I don't see that these are referenced any longer.  Remove?

Reply:
Yeah they shouldn't be, but I left them as a reference for future weapon creation, to give future coders a general idea.  They're probably better off being commented out, so I'll do that next update.

Reply:
On second thought, you're 100% right.  They should just be removed entirely.